### ISSUE TITLE
Commerce - Reviewing system requirements was not the experience I expected

### ISSUE THREAD

**Issue by reizero00** [liferay/liferay-learn#44]
Reviewing system requirements was not the experience I expected after going through the Installation Overview and Quick Start guide. I find it awkward that the System Requirements are already linked in the Installation Overview (to a PDF -- also awkward because of the context loss from moving to PDF view). In the screenshot below you can see System Requirements in the Installation Overview and on the doc navigator.

![Screenshot at 2019-11-15 14-28-37](https://user-images.githubusercontent.com/1261063/68980430-4bc68d80-07b5-11ea-8f6d-85a9c1914af7.png)

I would like to suggest removing the System Requirements section from the doc navigator completely and keep it within the Installation Overview document.

**Comment by jrhoun** [comment]
Yeah - it's a pretty rough experience. We'll get this fixed - _at the minimum_ it should probably open a new window instead of take you out of context.
